.github/workflows/ruby.yml

```diff
@@ -0,0 +1,43 @@
+name: Ruby CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
```

Comment:
We may want to change this to a specific major version for safety but I think this is fine to start and see if the thing runs:tm:

Reply:
Yep, I think that's a good idea. It looks like https://github.com/braintree/runbook/pull/50 resulted in some green builds, so we can start tweaking it some now.